scripts/openyurt_deployer/configs/knative.go

```diff
@@ -0,0 +1,39 @@
+package configs
+
+import "fmt"
+
+type KnativeConfigStruct struct {
+	KnativeVersion                       string
+	IstioVersion                         string
+	IstioDownloadUrlTemplate             string
+	IstioOperatorConfigUrl               string
+	MetalLBVersion                       string
+	MetalLBConfigURLArray                []string
+	LocalRegistryRepoVolumeSize          string
+	LocalRegistryVolumeConfigUrl         string
+	LocalRegistryDockerRegistryConfigUrl string
+	LocalRegistryHostUpdateConfigUrl     string
+	MagicDNSConfigUrl                    string
+	VHiveMode                            bool
+}
+
+var Knative = KnativeConfigStruct{
+	KnativeVersion:           "1.9.2",
```

Comment:
these configs should be taken from the vHive json config. there must be a consistent set of configs